package.json

```diff
@@ -112,12 +112,15 @@
     "enzyme-to-json": "^3.4.4",
     "eslint": "^7.19.0",
     "gh-pages": "^3.1.0",
-    "html-webpack-plugin": "^4.5.0",
+    "html-webpack-plugin": "^3.1.0",
     "jest": "^26.6.3",
     "lerna": "^3.16.4",
     "link-parent-bin": "^1.0.0",
     "loader-runner": "^4.1.0",
     "postcss": "^8.2.10",
+    "postcss-assets-webpack-plugin": "^4.0.0",
+    "postcss-custom-properties": "^10.0.0",
+    "postcss-loader": "^4.1.0",
```

Comment:
These are added to override the the older versions coming from terra-dev-site that we can't remove yet.